Library/Homebrew/global.rb

```diff
@@ -47,20 +47,3 @@ module Homebrew
 require "compat" unless ARGV.include?("--no-compat") || ENV["HOMEBREW_NO_COMPAT"]
 
 ORIGINAL_PATHS = ENV["PATH"].split(File::PATH_SEPARATOR).map { |p| Pathname.new(p).expand_path rescue nil }.compact.freeze
-
-HOMEBREW_INTERNAL_COMMAND_ALIASES = {
-  "ls" => "list",
-  "homepage" => "home",
-  "-S" => "search",
-  "up" => "update",
-  "ln" => "link",
-  "instal" => "install", # gem does the same
-  "rm" => "uninstall",
-  "remove" => "uninstall",
-  "configure" => "diy",
-  "abv" => "info",
-  "dr" => "doctor",
-  "--repo" => "--repository",
-  "environment" => "--env",
-  "--config" => "config"
-}
```

Comment:
@UniqMartin I think we can remove that line from `brew commands`; it hasn't been around long. I'll leave it here for now but the duplication makes me :sob: